### PR TITLE
[IMP] project: documentation overhaul 2

### PR DESCRIPTION
**Pull request by kbr-odoo**

### CODE REVIEW

File: project/advanced/feedback.rst [hunk 1]
```diff
@@ -1,108 +1,63 @@
...
+==============================
+Gather Feedback from Customers
+==============================
 
-Overview
-========
+Finding out what your clients think about the experience they have with your company, can inspire
+you to have insights on how to improve your product/service, adjusting your business to fit
+their needs. It shows that you value their opinions, and provides a reliable source of information
+to other consumers.
```
Comment (Lau-Odoo) [inline project/advanced/feedback.rst]:
```suggestion
Finding out what your clients think about the experience they have with your company can inspire
you to have insights on how to improve your product/service, adjusting your business to fit
their needs. It shows that you value their opinions and provides a reliable source of information
to other consumers.
```

File: project/advanced/feedback.rst [hunk 2]
```diff
@@ -1,108 +1,63 @@
...
-Project configuration
----------------------
+.. image:: media/rating_report.png
+   :align: center
+   :alt: View fo the customer rating report in Odoo Project
 
-Go to the **Project** application, in the project settings select the
-**Customer satisfaction** option.
+Clicking on the percentage of happy ratings over the last 30 days in the *Projects* overview,
+redirects you to your website with more rating details.
```
Comment (Lau-Odoo) [inline project/advanced/feedback.rst]:
```suggestion
Clicking on the percentage of happy ratings over the last 30 days in the *Projects* overview
redirects you to your website with more rating details.
```

File: project/advanced/feedback.rst (hunk 1 above)
Comment (jubodoo) [reply project/advanced/feedback.rst]: This wasn't resolved. You didn't accept his suggestion as far as I can see.

Comment (jubodoo) [reply project/advanced/feedback.rst]: @Lau-Odoo Tu peux recheck stp, de ce que je vois elle a cancel tes suggestions qui était juste.

Comment (kbr-odoo) [reply project/advanced/feedback.rst]: Sure. I'll accept it, even though I didn't in the beginning because I did not agree.

**Comment by kbr-odoo** [comment]
@Lau-Odoo previous not accepted comment was accepted now and pushed.
Thanks!

File: project/advanced/feedback.rst (hunk 1 above)
Comment (jubodoo) [reply project/advanced/feedback.rst]: As I said, never discard without explaining why it's frustrating for the reviewer. In this case, he was right, the sentence didn't make sense with the comma.

Comment (kbr-odoo) [reply project/advanced/feedback.rst]: Yes, ok. For minor suggestions, I didn't believe an explanation was needed. I apologize. Will always do it from now on. Thank you!

**Pull request merged**